fetch.bs

```diff
@@ -4506,6 +4506,10 @@ steps:
 
  <li><p>If <var>recursive</var> is false, then run the remaining steps <a>in parallel</a>.
 
+ <li><p>Let |raceResponse| be the result of <a for=/>lookup race response</a> given <var>request</var>.
```

Comment:
As far as I can tell https://w3c.github.io/ServiceWorker/#lookup-race-response-algorithm ends up waiting, which would delay this algorithm. Is that really the intent?

Reply:
Yes, but actually this wouldn't delay the algorithm. When |raceResponse| is not null, the corresponding request is at least in-flight status. The map in "Lookup Race Response" is a map only contains in-flight requests/responses. At this point in 4.1. Main fetch, the actual fetch is not performed yet, we expect waiting for the in-flight request is faster in most cases.